Dockerfile

```diff
@@ -31,7 +35,7 @@ RUN curl -sL https://github.com/nodenv/node-build/archive/master.tar.gz | tar xz
     rm -rf /tmp/node-build-master
 
 # Install application gems
-COPY Gemfile Gemfile.lock ./
+COPY Gemfile Gemfile.lock .ruby-version ./
```

Comment:
Is this necessary since the container already has the version baked in?

Reply:
Bundler is upset if I don't include it. I agree that it isn't great to have it duplicated, but as we are also supporting local development for the time being, I'm not a huge fan of any method to sync it either.

Reply:
Ah, what if you remove this line:
https://github.com/ABTech/tracker/blob/d9cb1c243f0540ce07857f3197bb24a29054d624/Gemfile#L3

I haven't seen this line in other Rails projects. It's probably a remnant of the past.

Reply:
Now that I think of it--I have run into this before and we should definitely remove this line. This line _enforces_ a specific Ruby version which is really annoying if you are developing on a machine with a different version (and without `rvm`).